synapse/api/auth.py

```diff
@@ -720,9 +720,9 @@ def get_access_token_from_request(request, token_not_found_http_status=401):
                     "Too many Authorization headers.",
                     errcode=Codes.MISSING_TOKEN,
                 )
-            parts = auth_headers[0].split(" ")
-            if parts[0] == "Bearer" and len(parts) == 2:
-                return parts[1]
+            parts = auth_headers[0].split(b" ")
+            if parts[0] == b"Bearer" and len(parts) == 2:
+                return parts[1].decode('ascii')
```

Comment:
can we change the docstring on this method to say it returns a `unicode` rather than a `str` ?

Reply:
fixed